### ISSUE TITLE
'Manage library execution' command missing

### ISSUE THREAD

**Issue by karansapolia** [microsoft/vscode-eslint#1419]
The README says that the extension provides `ESLint: Manage Library Execution` command through the VS Code command palette (cmd + shift + p on mac) but it seems like the command is missing. 

I was setting up [fastify](https://github.com/fastify/fastify/blob/main/docs/Guides/Contributing.md#setting-up-your-environment) on my machine and cannot perform the last step that involved running `ESLint: Manage Library Execution` from the command palette. Has the command been removed?

**Comment by dbaeumer** [comment]
Yes, the command has been removed in favor of VS Code's workspace trust. Will remove from the Readme.
